keyboards/madjax_macropad/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+ENCODER_ENABLE = yes
```

Comment:
Still no. `ENCODER_ENABLE` belongs at the keyboard level. This enables the hardware encoder feature. At the keymap level (this file) you want to enable the encoder configuration subsystem you are using (`ENCODER_MAP_ENABLE`).

Reply:
Sorry, the change to the json is a bit confusing. The docs all refer to the old way of doing stuff. Do y'all need help updating the docs? 

I think I got it this time.